skimage/morphology/convex_hull.py

```diff
@@ -2,6 +2,7 @@
 from itertools import product
 import numpy as np
 from scipy.spatial import ConvexHull
+from scipy.spatial.qhull import QhullError
```

Comment:
Before we merge this, let me double check if this `QhullError` is public API. SciPy has recently been renaming a lot of files to have underscores that were not intentionally public in the past...

Reply:
`scipy.spatial.qhull` is not public API as listed here:
https://docs.scipy.org/doc/scipy-1.7.0/reference/#api-definition
In fact it has now been renamed to `scipy.spatial._qhull` and unfortunately the `__all__` attribute on _qhull.pyx [does not include `QhullError`](https://github.com/scipy/scipy/blob/master/scipy/spatial/_qhull.pyx#L31).  Let me check with SciPy about this...

Reply:
I opened https://github.com/scipy/scipy/issues/14999 upstream to ask about this

Reply:
> 
> 
> I opened [scipy/scipy#14999](https://github.com/scipy/scipy/issues/14999) upstream to ask about this


[scipy/scipy#15003](https://github.com/scipy/scipy/pull/15003) is now merged into main that would make `QhullError` public API, great catch @grlee77 

Let me know if I need to do anything else.

Reply:
Thank you for quickly making the upstream PR @boazmohar!